submarine/pom.xml

```diff
@@ -130,6 +130,13 @@
     </dependency>
   </dependencies>
 
+  <repositories>
+    <repository>
+      <id>jerbrain</id>
+      <url>https://dl.bintray.com/jetbrains/pty4j/</url>
```

Comment:
`Pty4j` is the dependency used in the shell-terminal interpreter. The `pty4j` package should not be used in the submarine. Will it not compile now?

Reply:
It looks like due to my environment issue, will close this PR